frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -677,7 +677,7 @@ export type CodeEditorRef = {
 };
 
 export type ResourceYAMLEditorProps = {
-  initialResource: string | { [key: string]: any };
+  initialResource: K8sResourceKind;
```

Comment:
api change: EditYAML accepts only `K8sResourceKind` so this prop was typed wrong